components/scream/tests/uncoupled/p3/p3_standalone_output.yaml

```diff
@@ -7,28 +7,6 @@ FREQUENCY:
   OUT_N: 10
   OUT_OPTION: Steps
   OUT_MAX_STEPS: 1
-FIELDS:
-  Number of Fields: 8
-  field 1: T_mid
-  field 2: T_prev_micro_step
-  field 3: qv_prev_micro_step
-  field 4: eff_radius_qc
-  field 5: eff_radius_qi
-  field 6: micro_liq_ice_exchange
-  field 7: micro_vap_liq_exchange
-  field 8: micro_vap_ice_exchange
-#  field 9: bm
-#  field 10: nc
-#  field 11: ni
-#  field 12: nr
-#  field 13: qc
-#  field 14: qi
-#  field 15: qm
-#  field 16: qr
-#  field 17: qv
-#  field 7: nevapr - deprecated
-#  field 7: qr_evap_tend - deprecated
-#  field 11: precip_total_tend - deprecated
-#  field 4: lambda_qc - deprecated
-#  field 6: mu_qc - deprecated
+FIELDS: [T_mid, T_prev_micro_step, qv_prev_micro_step, eff_radius_qc, eff_radius_qi,
```

Comment:
We can add qv, qc, ... to this output file, now that you've fixed outputting tracers.

Reply:
Yes, that might add a bit of testing to the IO infrastructure.